docs/docs/draft/policies/README.md

```diff
@@ -378,6 +378,70 @@ In the `format` field, you can use [standard Envoy placeholders](https://www.env
 - `%KUMA_SOURCE_SERVICE%` - source service from which traffic is sent
 - `%KUMA_DESTINATION_SERVICE%` - destination service to which traffic is sent
 
+## Health Check
+
+The goal of Health Checks is to minimize the number of failed requests due to temporary unavailability of a target endpoint.
+By applying a Health Check policy you effectively instruct a dataplane to keep track of health statuses for target endpoints.
+Dataplane will never send a request to an endpoint that is considered "unhealthy".
+
+Since pro-active health checking might result in a tangible extra load on your applications,
+Kuma also provides a zero-overhead alternative - "passive" health checking.
+In the latter case, a dataplane will be making decisions whether target endpoints are healthy based on "real" requests
+initiated by your application rather than auxiliary requests initiated by the dataplanes itself.
+
+As usual, `sources` and `destinations` selectors allow you to fine-tune to which `Dataplanes` the policy applies (`sources`)
+and what endpoints require health checking (`destinations`).
+
```

Comment:
Can we include a link to https://www.envoyproxy.io/docs/envoy/v1.5.0/intro/arch_overview/outlier if anyone wants to dig a little bit more into it?

Maybe it's also worth of mentioning that this HC is based on checking TCP connection.